game/views.py

```
@@ -628,7 +628,21 @@ def verify_otp(request):
         remaining_time = max(0, 60 - elapsed)
-    return render(request, 'game/verify_otp.html', {'remaining_time': remaining_time})
+
+    masked_email = None
+    try:
+        user = User.objects.get(id=user_id)
+        email = user.email
+        local, domain = email.split('@')
+        masked_local = local[0] + '*' * (len(local) - 1)
+        masked_email = f"{masked_local}@{domain}"
+    except Exception:
+        pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)